src/lib/components/file_tree/FileTree.svelte

```diff
@@ -85,13 +86,33 @@
 			}
 		},
 	});
+
+	function file_input_change(path: string) {
+		return (e: Event) => {
+			const files = [...((e.currentTarget as HTMLInputElement)?.files ?? [])];
+			handle_files(files, files_options(path));
+		};
+	}
+
+	function get_upload_handler(path = base_path) {
+		return () => {
+			const file_input = document.createElement('input');
+			file_input.type = 'file';
+			file_input.multiple = true;
+			file_input.addEventListener('change', file_input_change(path));
```

Comment:
Do we have to remove the event listener after completion to prevent a memory leak?

Reply:
Mmm in theory the whole input should be garbage collected after the thing because noone hold a reference to it...but better be safe than sorry...i'll fix this and push it back up